source/beamline/BioXAS/BioXASSideBeamline.cpp

```diff
@@ -268,7 +266,7 @@ void BioXASSideBeamline::setupComponents()
 	// Detector stage.
 
 	detectorStageLateral_ = new CLSMAXvMotor("SMTR1607-6-I22-16", "SMTR1607-6-I22-16", "SMTR1607-6-I22-16", true, 0.05, 2.0, this, ":mm");
-	connect( detectorStageLateral_, SIGNAL(connected(bool)), this, SLOT(updateConnected()) );
+	addDetectorStageLateralMotor(detectorStageLateral_);
```

Comment:
Yes, I think we should! If it's not there, then I think I left it out by mistake. I'll take a look. Good catch!